ktor-server/ktor-server-test-base/jsAndWasmShared/src/io/ktor/server/test/base/EngineTestBase.jsAndWasmShared.kt

```diff
@@ -101,7 +125,8 @@ actual constructor(
         // we start it on the global scope because we don't want it to fail the whole test
         // as far as we have retry loop on call side
         val starting = GlobalScope.async {
-            server.start(wait = false)
+            server.startSuspend(wait = false)
```

Comment:
Should we make an extension function here like `server.listeningPorts()`

Reply:
I guess that the commend was for this line:
```
_port = server.engine.resolvedConnectors().first().port
```

I don't think, that we do really need this, as this is rather specific and mostly needed in tests where we need to launch both server and client (e.g ktor, or in projects like rsocket-kotlin)